openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -28,19 +24,19 @@ default_availability_zone = {{.Values.global.default_availability_zone}}
 rpc_response_timeout = {{ .Values.rpc_response_timeout | default .Values.global.rpc_response_timeout | default 60 }}
 rpc_workers = {{ .Values.rpc_workers | default .Values.global.rpc_workers | default 1 }}
 
-wsgi_default_pool_size = {{ .Values.wsgi_default_pool_size | default .Values.global.wsgi_default_pool_size | default 100 }}
 sync_power_state_pool_size = {{ .Values.sync_power_state_pool_size | default 500 }}
 sync_power_state_interval = {{ .Values.sync_power_state_interval | default 1200 }}
 sync_power_state_unexpected_call_stop = false
 
 prepare_empty_host_for_spawning_interval = 600
 
-dhcp_domain = openstack.{{ required ".Values.global.region is missing" .Values.global.region }}.{{ required ".Values.global.tld is missing" .Values.global.tld }}
-
```

Comment:
I think we still need that and it was undeprecated in a later release, because that's how we set the domain-name.

Reply:
OK.
Seems it gets back in Train under [api].

https://github.com/openstack/nova/blob/886b0a5d748ae1deda3a039734f831d7c0cf0476/releasenotes/notes/undeprecate-dhcp_domain-opt-77c9154c5b06e0ff.yaml#L2-L8